src/contest_manager.py

```diff
@@ -48,10 +49,9 @@ def import_contest(path):
             # copy the statement directory into the statement folder
             statementdir = os.path.join(Config.statementdir, task)
             taskdir = os.path.join(path, task)
-            if not os.path.isdir(statementdir):
-                if os.path.exists(statementdir):
-                    shutil.rmtree(statementdir)
-                shutil.copytree(os.path.join(taskdir, "statement"), statementdir)
+            if os.path.isdir(statementdir):
+                shutil.rmtree(statementdir)
+            shutil.copytree(os.path.join(taskdir, "statement"), statementdir)
```

Comment:
Something bad...

Before there was an other error, note that `shutil.rmtree` can only remove directories!
An error is raised if you try to remove a file
```NotADirectoryError: [Errno 20] Not a directory: '/tmp/foobar.txt'```

If `statementdir` is a file we have to remove it. The question now is: if it's a directory should we keep it?